app/graph/mutations/api_key_mutations.rb

```diff
@@ -0,0 +1,36 @@
+module ApiKeyMutations
+  MUTATION_TARGET = 'api_key'.freeze
+  PARENTS = ['team'].freeze
+
+  module CreateFields
+    extend ActiveSupport::Concern
+
+    included do
+      argument :title, GraphQL::Types::String, required: false
+      argument :description, GraphQL::Types::String, required: false
+    end
+  end
+
+  class Create < Mutations::CreateMutation
+    include CreateFields
```

Comment:
If this is the only place we're using `CreateFields`, it doesn't need to a module, right? We could just add the `argument` calls straight in this class body.